extensions/sonos/package.json

```diff
@@ -67,6 +70,30 @@
       "mode": "no-view"
     }
   ],
+  "preferences": [
+    {
+      "name": "volumeStep",
+      "description": "Adjust the increment change in volume levels.",
+      "type": "dropdown",
+      "required": false,
+      "title": "Volume Step",
+      "default": "1",
```

Comment:
The previous default was 2, just a note. I don't have a strong opinion, I'm fine with 1 also.

Reply:
Totally agree! I may have overcomplicated the labels. This is a good example where less is more. I changed the labels to Small, Medium, and Large, and set Small/Default to 2.